decoder_test.go

```diff
@@ -294,4 +294,46 @@ func TestDataMultipleAtendee(t *testing.T) {
 		t.Errorf("Atendee list should be %s", att)
 	}
 
-}
\ No newline at end of file
+}
+
+var dataMultiline = `BEGIN:VCALENDAR
+BEGIN:VEVENT
+ORGANIZER:Mailto:B@example.com
+ATTENDEE;
+ ROLE=CHAIR;
+  PARTSTAT
+   =ACCEPTED;CN=
+    BIG A
+	 :Mailto:A@example.com
+ATTENDEE;RSVP=TRUE;CUTYPE=INDIVIDUAL;
+ CN=B:Mailto:B@example.com
+ATTENDEE;RSVP=TRUE;CUTYPE=INDIVIDUAL;CN=C:Mailto:C@example.com
+DTSTAMP:19970611T193000Z
+DTSTART:19970701T190000Z
+DTEND:19970701T193000Z
+UID:calsrv.example.com-873970198738777@example.com
+SUMMARY: Pool party
+END:VEVENT
+END:VCALENDAR`
+
+func TestMulitine(t *testing.T) {
+
+	d := goics.NewDecoder(strings.NewReader(dataMultiline))
+	consumer := EventsA{}
+	err := d.Decode(&consumer)
+	if err != nil {
+		t.Error("Error decoding events")
+	}
+	if len(consumer) != 1 {
+		t.Error("Wrong size of consumer list..")
+	}
+
+	if len(consumer[0].Attendees) != 3 {
+		t.Errorf("Wrong list of atendees detectet %d", len(consumer[0].Attendees))
+	}
+
+	att := consumer[0].Attendees[0]
+	if att != "Mailto:A@example.com" {
+		t.Errorf("Atendee list should be %s", att)
```

Comment:
Another typo. I realize there were existing typos in this file already. While those could be updated in this PR or even a separate one, I definitely don't think this PR should add new typos.